pegasus/migrations/093_add_hashed_email_to_contacts.rb

```diff
@@ -0,0 +1,30 @@
+require 'digest/md5'
+
+Sequel.migration do
+  up do
+    add_column :contacts, :hashed_email, String, null: false
+
+    batch_update do |contact|
+      self[:contacts].where(id: contact[:id]).update(hashed_email: Digest::MD5.hexdigest(contact[:email].downcase))
```

Comment:
Note that this line, since `email` is always non-null, is both safe to execute and guaranteed to populate the `hashed_email` column.
```
mysql> SHOW CREATE TABLE contacts;
| contacts | CREATE TABLE `contacts` (
  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `email` varchar(254) NOT NULL,...
```
```
SELECT COUNT(0) FROM contacts WHERE email IS NULL;
+----------+
| COUNT(0) |
+----------+
|        0 |
+----------+
````